src/components/pagination/pager-numeric.js

```diff
@@ -0,0 +1,159 @@
+import '@brightspace-ui/core/components/button/button-icon.js';
+import '@brightspace-ui/core/components/inputs/input-number.js';
+import '@brightspace-ui/core/components/inputs/input-text.js';
+import { css, html, LitElement } from 'lit';
+import { formatNumber } from '@brightspace-ui/intl';
+import { LocalizeLabsElement } from '../localize-labs-element.js';
+import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
+import { selectStyles } from '@brightspace-ui/core/components/inputs/input-select-styles.js';
+
+class PagerNumeric extends RtlMixin(LocalizeLabsElement(LitElement)) {
+
+	static get properties() {
+		return {
+			maxPageNumber: { type: Number, attribute: 'max-page-number', reflect: true },
+			pageNumber: { type: Number, attribute: 'page-number', reflect: true },
+			pageSize : { type: Number, attribute: 'page-size', reflect: true },
```

Comment:
It looks like there is nothing enforcing the `pageSize` is a value in the array. If it's not valid, we could consider using a [custom dependency validator](https://github.com/BrightspaceUI/core/tree/main/mixins/property-required#dependent-properties) to catch it.

That scenario is maybe just an incorrect configuration that wouldn't cause any issues, so not a big deal at this point in time. Don't feel obligated to do that.

Reply:
I think I'm going to omit this validation for the time being. When I tried implementing it, it kept throwing errors because neither `pageSize` nor `pageSizes` are Strings, and `only String properties can be required` according to the error that was being thrown.